src/lib/qnio/nio_server.c

```diff
@@ -452,14 +452,16 @@ qns_server_start(char *node, char *port)
                             {
                                 nioDbg("SSL_error is want read or want write %d", ret);
                                 /* put fd in epoll loop to try again */
-                                event.events = EPOLLIN;
+                                event.events = EPOLLIN | EPOLLOUT;
```

Comment:
This causes the event loop to spin, burning CPU.  Imagine OpenSSL is waiting to receive data.  EPOLLOUT causes epoll_wait() to return immediately because the socket is writable.  OpenSSL won't be able to make any progress though since it's waiting to receive (not send).  Therefore this change causes the process to consume CPU.

As I wrote in the previous review comment: SSL_ERROR_WANT_READ -> EPOLLIN,  SSL_ERROR_WANT_WRITE -> EPOLLOUT.